.circleci/config.yml

```diff
@@ -6,7 +6,7 @@ general:
 jobs:
   build:
     docker:
-      - image: circleci/node:8
+      - image: circleci/node:8-browsers
```

Comment:
Not sure about this, `puppeteer` package will install an own browser. Do we need to switch images?

Reply:
Yes.
Puppeteer installs its own Chrome, but it still depends on some shared libraries which are not installed in the original image:
```
[02:10:55] 'perf:run' errored after 174 ms
[02:10:55] Error: Failed to launch chrome!
/home/circleci/project/node_modules/puppeteer/.local-chromium/linux-609904/chrome-linux/chrome: error while loading shared libraries: libXtst.so.6: cannot open shared object file: No such file or directory
```